src/gluonts/env.py

```diff
@@ -21,8 +21,8 @@ class Environment(Settings):
     # returning an output. This parameter is intended to catch infinite loops
     # or inefficiencies, when transformations never or rarely return
     # something.
-    max_idle_transforms: int = os.environ.get(
-        "GLUONTS_MAX_IDLE_TRANSFORMS", "100"
+    max_idle_transforms: int = int(
```

Comment:
I think this should be handled in `settings.py` by setting `validate_all` in the `_Config`.

Reply:
Mhh ok, a little confusing tbh but I'll change it^^

Reply:
What exactly is confusing?

Reply:
It's not really obvious how the types are converted (at least I didn't find out at a glance)

Reply:
All fields are converted using `pydantic`. Actually, this should just work as it is.